recipesearch/src/Recipes/SearchResults/SearchResults.js

```diff
@@ -1,17 +1,21 @@
 /* eslint-disable jsx-a11y/img-redundant-alt */
```

Comment:
So it was complaining because in the alt, I put the word picture and it was complaining that in an alt I should not put anything related to picture, photo or image.
The above line was to switch the error off but I have since removed it as well as the word picture in the alt